src/OneDriveSdk.WindowsForms/Web/FormsWebDialog.cs

```diff
@@ -52,6 +52,15 @@ public FormsWebDialog()
                 return null;
             }
 
+            bool isSignOutRequest =
+                requestUri.AbsoluteUri.StartsWith(Constants.Authentication.MicrosoftAccountSignOutUrl,
+                    StringComparison.OrdinalIgnoreCase) ||
+                requestUri.AbsoluteUri.StartsWith(Constants.Authentication.ActiveDirectorySignOutUrl,
+                    StringComparison.OrdinalIgnoreCase);
+
+            this.ShowInTaskbar = !isSignOutRequest;
+            this.WindowState = !isSignOutRequest ? FormWindowState.Normal : FormWindowState.Minimized;
```

Comment:
Could you flip the logic here? For if/else statements it looks a bit cleaner to start with the affirmative check?